packaging/installer/netdata-updater.sh

```diff
@@ -153,7 +159,13 @@ newer_commit_date() {
     commit_date="1970-01-01T00:00:00Z"
```

Comment:
Shouldn't this be something very far in the future, so that the default behaviour is to always overwrite the updater if something went wrong?

Reply:
Actually, yes, I somehow got the logic backwards here.

Reply:
Updated now to default to a date long past when we will all be dead for the last commit date. If Netdata is still around in seven thousand nine hundred and seventy nine years and this code is still being used, our distant descendants can deal with any issues this might cause then (alongside all the other issues that may exist at that time due to most ISO 8601 date parsing code assuming a four digit year).